src/main/resources/db/mysql/petclinic_db_setup_mysql.txt

```diff
@@ -22,7 +22,14 @@
    scripts (or set "spring.datasource.initialization-mode=always" the first time you run the app).
 
 3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
-   line, but any way that sets that property in a Spring Boot app should work).
+   line, but any way that sets that property in a Spring Boot app should work). For newer versions
```

Comment:
As I've stated previously, I don't think we should add a note about "newer version". The attribute I've shared with "old" and "new" version irrespective of forking and is the recommended way to use the plugin. I would rather remove the content in parenthesis completely as it implies you can pass a system property that way and it will work.